client/blocks/reader-full-post/index.jsx

```diff
@@ -369,84 +346,92 @@ export class FullPostView extends React.Component {
 							/> }
 
 					</div>
-					<article className="reader-full-post__story" ref="article">
-						<ReaderFullPostHeader post={ post } referralPost={ referralPost } />
-
-						{ post.featured_image &&
-							! isFeaturedImageInContent( post ) &&
-							<FeaturedImage src={ post.featured_image } /> }
-						{ isLoading && <ReaderFullPostContentPlaceholder /> }
-						{ post.use_excerpt
-							? <PostExcerpt content={ post.better_excerpt ? post.better_excerpt : post.excerpt } />
-							: <EmbedContainer>
-									<AutoDirection>
-										<div
-											className="reader-full-post__story-content"
-											dangerouslySetInnerHTML={ { __html: post.content } }
+					<Emojify>
```

Comment:
yay, much nicer than doing the parser work via lifecycle methods! Emojify adds a div, did that alter the reader display at all?